pkg/json.go

```diff
@@ -77,14 +77,14 @@ func (s jsonFloatScore) MarshalJSON() ([]byte, error) {
 
 // JSONScorecardResultV2 exports results as JSON for new detail format.
 //
-//nolint:govet
+
 type JSONScorecardResultV2 struct {
```

Comment:
note: the blank line breaks the godoc. Please change to either:
```go
// JSONScorecardResultV2 exports results as JSON for new detail format.
//
type JSONScorecardResultV2 struct {
```

but preferably:
```go
// JSONScorecardResultV2 exports results as JSON for new detail format.
type JSONScorecardResultV2 struct {
```